tests/generic_config_updater/test_aaa.py

```diff
@@ -15,6 +15,11 @@
 AAA_CATEGORY = ["authentication", "authorization", "accounting"]
 DEFAULT_TACACS_SERVER = "100.127.20.21"
 
+
+@pytest.fixture(autouse=True, scope="module")
+def avoid_long_running_test():
+    pytest.skip("Test costs too much time. Temp skip for now.")
```

Comment:
Official document gives a short solution with `allow_module_level`
https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skipping-test-functions

Reply:
Found another way to add marker for skip.